sys/linux/filesystem.txt

```diff
@@ -31,8 +31,21 @@ fsconfig$FSCONFIG_CMD_RECONFIGURE(fd fd_fscontext, cmd const[FSCONFIG_CMD_RECONF
 fsmount(fs_fd fd_fscontext, flags flags[fsmount_flags], attr_flags flags[fsmount_attr_flags]) fd
 move_mount(from_dfd fd_dir[opt], from_pathname ptr[in, filename], to_dfd fd_dir[opt], to_pathname ptr[in, filename], flags flags[move_mount_flags])
 open_tree(dfd fd_dir[opt], filename ptr[in, filename], flags flags[open_tree_flags]) fd
+mount_setattr(dfd fd_dir[opt], filename ptr[in, filename], flags flags[mount_setattr_flags], args ptr[in, mount_attr], size bytesize[args])
 
 open_tree_flags = AT_EMPTY_PATH, AT_NO_AUTOMOUNT, AT_RECURSIVE, AT_SYMLINK_NOFOLLOW, OPEN_TREE_CLONE, OPEN_TREE_CLOEXEC
+
+mount_setattr_flags = AT_EMPTY_PATH, AT_NO_AUTOMOUNT, AT_RECURSIVE, AT_SYMLINK_NOFOLLOW
+
+mount_attr {
+	attr_set	flags[mount_attr_flags, int64]
+	attr_clr	flags[mount_attr_flags, int64]
+	propagation	flags[mount_attr_propagation_flags, int64]
```

Comment:
All patches I can find define this struct differently:
```
+struct mount_attr {
+	__u64 attr_set;
+	__u64 attr_clr;
+	__u32 propagation;
+	__u32 atime;
+};
```
https://lore.kernel.org/lkml/20200714161415.3886463-5-christian.brauner@ubuntu.com/
Where should I be looking?

Reply:
Sorry, this is ancient. Just look here:
https://git.kernel.org/pub/scm/linux/kernel/git/brauner/linux.git/tree/include/uapi/linux/mount.h?h=idmapped_mounts#n127